src/components/FeaturedTopics/FeaturedTopics.css

```diff
@@ -0,0 +1,45 @@
+@import '../variables.css';
+
+.root {
+  --lightBg: rgb(256, 256, 256);
+
+  background: rgb(237, 237, 237);
+}
+
+.center {
+  margin: auto;
+  width: 50%;
+  text-align: center;
+  text-rendering: optimizeLegibility;
+}
+
+.pushBottom {
+  padding-bottom: 5em;
+}
+
+.pushTop {
+  padding-top: 2.5em;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  padding: 1em;
+  overflow: auto;
+}
+
+.floatingBox {
+  width: 20%;
+  height: 60%;
+  margin: 1em;
+  padding: 1em;
+}
+
+.small {
+  font-size: 0.75em;
+}
+
+.light {
+  background-color: var(--lightBg);
```

Comment:
I am not sure why we should put all the colors in variable. Is this what you originally intended?

Reply:
all colours should bein the variable file and we should add rules in our linter to ensure that colours are not in .css